spec/app/dependency_file_updaters/node_spec.rb

```diff
@@ -63,11 +63,12 @@
     end
   end
 
-  describe "#updated_shrinkwrap" do
-    subject(:file) { updater.updated_shrinkwrap }
+  describe "#updated_yarn_lock" do
+    subject(:file) { updater.updated_yarn_lock }
+    # This currently fails on Travis :(
```

Comment:
Comment still relevant? If so, would be good to give some context as to why it fails.